utilities_common/common.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python -u
+# -*- coding: utf-8 -*-
+
+__all__ = ['log_info', 'log_error', 'log_warning', 'log_debug',
+        'do_exit', 'get_configdb_data', 
+        'AbbreviationGroup', 'get_hostname']
+
+import sys
+import os
+import syslog
+import click
+import inspect
+from swsssdk import ConfigDBConnector
+
+SYSLOG_IDENTIFIER = "config"
+
+def get_hostname():
+    return os.uname()[1]
+
+# ========================== Syslog wrappers ==========================
+
+def _log_msg(lvl, print_msg, fname, ln, msg):
+    syslog.openlog(SYSLOG_IDENTIFIER)
+    syslog.syslog(lvl, msg)
+    syslog.closelog()
+
+    if (print_msg and
+            (lvl == syslog.LOG_DEBUG or
+                sys.flags.interactive or
+                sys.flags.debug or
+                sys.flags.verbose)):
+        print("{}: {}:{}: {}".format(SYSLOG_IDENTIFIER, fname, ln, msg))
+
+
+def log_info(msg, print_msg=False):
+    _log_msg(syslog.LOG_INFO, print_msg, inspect.stack()[1][1], inspect.stack()[1][2], msg)
+
+
+def log_error(msg, print_msg=False):
+    _log_msg(syslog.LOG_ERR, print_msg, inspect.stack()[1][1], inspect.stack()[1][2], msg)
+
+
+def log_warning(msg, print_msg=False):
+    _log_msg(syslog.LOG_WARNING, print_msg, inspect.stack()[1][1], inspect.stack()[1][2], msg)
+
+
+def log_debug(msg, print_msg=False):
+    _log_msg(syslog.LOG_DEBUG, print_msg, inspect.stack()[1][1], inspect.stack()[1][2], msg)
```

Comment:
These syslog wrappers are not needed, as common logging functionality is now handled by sonic-py-common. Please remove.

Reply:
done.